BUILD.bazel

```diff
@@ -11,3 +11,24 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+load("@bazel_gazelle//:def.bzl", "gazelle", "gazelle_binary", "DEFAULT_LANGUAGES")
+
+# gazelle:prefix github.com/bazelbuild/rules_sass
+gazelle(
+    name = "gazelle",
+    gazelle = ":gazelle_sass",
+)
+
+# In your project you should define your own Gazelle binary. Follow the
+# instructions at
```

Comment:
we don't want our users to compile Go code, no. But we haven't tackled the details of how that works along with Gazelle build rules yet.

Reply:
I'm not sure how to proceed on this.

Reply:
For the whole npm ecosystem we don't want our users to observe a dependency on go code since they have to download a whole Go sdk. Our approach for other go code has been to ship three binaries to npm. I imagine we'll eventually do the same for BUILD file generation - so you'd get something like `bazel run @npm//@bazel/gazelle/bin:gazelle` and it would not require any dependencies in WORKSPACE